common/source/docs/common-uavcan-adapter-node.rst

```diff
@@ -60,8 +60,36 @@ An AP_Periph UAVCAN firmware supports these UAVCAN features:
 Setup
 ======
 
-Rangefinder
-------------
+AP_Periph f303-universal Firmware
+---------------------------------
+
+ The f303-Universal Firmware has the ability to be used for several devices but only one can be
+ enabled to use the single uart.
+ Once uploaded the default device is set to GPS, to use another device such as Rangefinder the
+ GPS has to be turned off and Rangefinder or other device enabled. To achieve this either use
+ UAVCAN GUI or MissionPlanner SLCAN. (link here). Once connected set params i.e. for TFmini_S/Plus rangefinder
```

Comment:
use UAVCAN GUI or Mission Planner SLCAN to set the UAVCAN adpater's internal parameters.

fyi https://uavcan.org/GUI_Tool/Overview/ is UAVCAN GUI link
ref: for MP SLCAN wiki page is <common-mp-slcan>

Reply:
Thanks for the fyi, I was unsure of the correct usage